### PR TITLE
mtx: add commands

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/mtx/commands/uplink/getRatePlanInfo.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/**
+ * Downlink command to get device rate plan information.
+ *
+ * The corresponding downlink command: `GetRatePlanInfo`.
+ *
```
Comment (DarkPark) [inline src/mtx/commands/uplink/getRatePlanInfo.ts:5]: @packageDocumentation

File: src/mtx/commands/uplink/getSaldoParameters.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+export const id: types.TCommandId = 0x2e;
+export const name: types.TCommandName = 'getSaldoParameters';
+export const headerSize = 2;
+export const maxSize = 37;
+export const accessLevel: types.TAccessLevel = READ_ONLY;
+
+export const examples: command.TCommandExamples = {
+    'default response': {
+        id,
+        name,
```
Comment (DarkPark) [inline src/mtx/commands/uplink/getSaldoParameters.ts:50]: headerSize

and other places

File: src/mtx/commands/downlink/runTariffPlan.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+export const examples: command.TCommandExamples = {
+    'simple request': {
+        id,
+        name,
+        headerSize,
+        maxSize,
+        accessLevel,
+        parameters: {tariffTable: 5},
+        bytes: [0x06, 0x01, 0x05]
```
Comment (DarkPark) [inline src/mtx/commands/downlink/runTariffPlan.ts:52]: multiline with header/body separation?

File: src/mtx/commands/downlink/setSaldo.ts [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        minutes: types.TUint8;
+    };
+
+    /**
+     * Enter new prepayment.
+     */
+    saldoNew: types.TInt32;
+
+    /**
+     * Saldo value obtained through {@link GetSaldo}/{@link GetSaldoResponse}.
```
Comment (DarkPark) [inline src/mtx/commands/downlink/setSaldo.ts:54]: do links work here? there are no imports /

File: src/mtx/commands/uplink/setSaldo.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
+/**
+ * Downlink command to set device current saldo information.
+ *
+ * The corresponding downlink command: `SetSaldo`.
```
Comment (DarkPark) [inline src/mtx/commands/uplink/setSaldo.ts:4]:
```suggestion
 * The corresponding downlink command: `setSaldo`.
```

File: src/mtx/commands/uplink/getRatePlanInfo.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/mtx/commands/uplink/getRatePlanInfo.ts:5]: fixed

File: src/mtx/commands/uplink/getSaldoParameters.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/mtx/commands/uplink/getSaldoParameters.ts:50]: fixed

File: src/mtx/commands/uplink/setSaldo.ts (hunk 5 above)
Comment (DeadByElpy) [reply src/mtx/commands/uplink/setSaldo.ts]: fixed in all commands

File: src/mtx/commands/downlink/setSaldo.ts (hunk 4 above)
Comment (DeadByElpy) [reply src/mtx/commands/downlink/setSaldo.ts]: imports added

**Review by DarkPark — APPROVED** [review]

**Pull request merged**